torch/_inductor/kernel/bmm.py

```diff
@@ -87,9 +86,14 @@ def bmm_grid(b, m, n, meta):
 aten_baddbmm = ExternKernelChoice(torch.baddbmm, "at::baddbmm_out")
 
 
-@register_lowering(aten.bmm)
+@L.register_lowering(aten.bmm)
```

Comment:
Nit: I guess you can still keep `register_lowering` without the need of changing this line.

Reply:
Modify it because we need to import many functions from lowering now, e.g. register_lowering/unsqueeze/mul/sum_.

Reply:
I mean you can still keep the import of `register_lowering` in parallel with importing `lowering`. Not a big deal but just wanted to keep this annotation consistent with other code.